Sources/Plot/API/HTMLAttributes.swift

```diff
@@ -401,7 +459,7 @@ public extension Attribute where Context == HTML.IFrameContext {
     /// Assign whether to grant the iframe full screen capabilities.
     /// - parameter allow: Whether the iframe should be allowed to go full screen.
     static func allowfullscreen(_ allow: Bool) -> Attribute {
-        Attribute(name: "allowfullscreen", value: String(allow))
+        Attribute(name: "allowfullscreen", value: nil, ignoreIfValueIsEmpty: false)
```

Comment:
This will always add the `allowfullscreen` attribute even if `allow = false`.

Reply:
Bugs! Great catch. Fixed in fddf87c8785a23fabf9821161f0682ddedabede7 and also added a test so it can't regress.